### DISCUSSION TITLE
Add importing of eggs through artisan.

### DISCUSSION THREAD

**Discussion by YajTPG** [pterodactyl/panel#2816] [Feature Requests]
Eg. command could be like :- `php artisan p:egg:import <jsonfile>`, `php artisan p:egg:delete <EggID>`, etc. Not sure if this is possible but it would be a great idea! (It's faster than manually importing a large number of eggs in most cases)

**Comment by ElijahRus250** [comment]
I like that idea! Would be super useful!!

**Comment by DaneEveritt** [comment]
At least in the meantime you can actually just drop your egg files into `database/seeds/eggs` and re-run the seeding command and it should load them up. However, at the end of the day importing eggs is generally not a frequent enough action to really warrant putting much time into that (IMO).

**Reply by ElijahRus250** [reply]
hmm, yea I guess you really only add an egg every once and a while and the online guide to edit the flags and file is nice. Makes sense.
